app/uploaders/simple_forms_api/form_remediation/uploader.rb

```diff
@@ -31,8 +31,13 @@ def store_dir
         @directory
       end
 
-      def get_s3_link(file_path)
-        s3_obj(file_path).presigned_url(:get, expires_in: 30.minutes.to_i)
+      def get_s3_link(file_path, filename = nil)
+        filename ||= File.basename(file_path)
+        s3_obj(file_path).presigned_url(
+          :get,
+          expires_in: 30.minutes.to_i,
+          response_content_disposition: "attachment; filename=\"#{filename}\""
+        )
```

Comment:
do we want an error if the s3 obj can't be found?

Reply:
Error handling is happening under the hood with AWS S3 Client and CarrierWave